packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -741,6 +741,17 @@ export const instructionTypeDefs = /* GraphQL */ `
         signers: [Address]
     }
 
+    """
+    SplToken-2022: InitializeConfidentialTransferMint instruction
+    """
+    type SplTokenInitializeConfidentialTransferMint implements TransactionInstruction {
+        programId: Address
+        auditorElgamalPubkey: Address
+        authority: Account
+        mint: Account
+        autoApproveNewAccounts: Boolean
```

Comment:
tiny nit: The pattern I've been following for schema fields is to put `programId` at the top, since it's part of the interface, and then sort the remaining fields alphabetically.